engines/experiment/docs/concepts.md

```diff
@@ -4,26 +4,16 @@
-
-Type        | Definition
-------------|-----------
-Filter      | This is used to filter the incoming request to decide whether or not it is in the experiment
-Unit        | Experiment Unit(s) whose values, when varied, have an impact on the Treatment generated
-Unsupported | The experiment engine does not distinguish between the variable types
-
+* **Experiment Variables** - Input values that have an impact on the 'Treatment' generated by the Experiment Engine. These are retrieved from the incoming request header or payload and applied when running the experiment.
 * **'Running' an Experiment** - Generating a Treatment for the given variables, by processing the Turing Experiment.
-* **Client** - In Experiment Engines that implement access control, someone with appropriate permissions to run an experiment
+* **Client** - In Experiment Engines that implement access control, someone with appropriate permissions to run an experiment.
 * **Variant** - A set of static configurations (as with A/B experiments), that would be selected at random, to be applied to the current request.
 * **Treatment** - The configuration to be applied to the current request. This may be one of the static Variants, if exists, or dynamically generated (as in the case of experiment engines like PlanOut).
 
 ## Workflow
 
-When creating a router, Turing users may choose to configure an experiment on an available experiment engine (the [Developer Guide](./developer_guide.md) has info on adding experiment engines to Turing). With every incoming request to Turing, a call will be made to the experiment engine to retrieve the treatment, which will be passed to the Ensembling phase, along with the response from each model route. To generate this treatment, an experiment engine typically relies on the following pieces of information (these are enabled by the experiment engine, according to its requirements).
-* **Client Info** - If the experiment engine supports basic auth / authorization, this specifies the credentials used to access the APIs, to generate the treatment.
-* **Experiment(s)** - One or more experiments that must be run for every request.
-* **Variables** - Parameters for running the experiment(s), to be retrieved from the request header / payload.
-
-Turing users will configure these properties through the Turing app, when they set up their routers. More details may be found under [Configure Experiment Engine](../../../docs/markdowns/creating-router/configure_rules_engine.md).
+When creating a router, Turing users may choose to configure an experiment on an available experiment engine (the [Developer Guide](./developer_guide.md) has info on adding experiment engines to Turing). When the router is deployed, with every incoming request to the router, a call will be made to the experiment engine using the following information.
```

Comment:
Nitpick:

```suggestion
When creating a router, Turing users may choose to configure an experiment on an available experiment engine (the [Developer Guide](./developer_guide.md) has info on adding experiment engines to Turing). When the router is deployed, with every incoming request to the router, a call will be made to the experiment engine with the following information:
```

Reply:
Oh I intentionally put using because "with" makes it look like that info is passed down in the API call. Rather, it may be implicitly made use of in some cases.